SCREENSHOTS.md

```diff
@@ -78,3 +78,44 @@ Current implementation of the utility is based on following external libraries:
 - [pngjs](https://github.com/lukeapage/pngjs): For image read/write to disk
 
 
+# Screenshot reporting
```

Comment:
Picky one: Shouldn't this title be at the same level of "generate a screenshot" or "screenshot comparison"? If that's the case, I would use 2 hashes for it